nuget.config

```diff
@@ -8,6 +8,7 @@
 		<clear />
 		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
 		<add key="Microsoft Health OSS" value="https://microsofthealthoss.pkgs.visualstudio.com/FhirServer/_packaging/Public/nuget/v3/index.json" />
+		<add key="Microsoft MedTech OSS" value="https://microsofthealthoss.pkgs.visualstudio.com/MedTech/_packaging/MedTechOSS/nuget/v3/index.json" />
```

Comment:
I'm guessing this is needed to pull in Microsoft.Health.Common.
Out of scope for this PR, but I think we should move common components (including telemetry) to a different feed. It is not intuitive as to why we need MedTech components in here otherwise. 
But that is a larger refactor, just wanted to call out.

Reply:
I agree. and I also agree that this should be handled in another scope of work.